cloud/errors.go

```diff
@@ -50,3 +55,33 @@ func (e cpiError) Message() string {
 func (e cpiError) OkToRetry() bool {
 	return e.cmdError.OkToRetry
 }
+
+func mapsToNotImplementedError(method string, cmdError CmdError) bool {
+	matched, _ := regexp.MatchString("^Method .+ not supported in photon CPI", cmdError.Message)
```

Comment:
may be just ignore any error for now? in 6 months we can be a bit more specific.